core/editor/loader/editor-base-loader.php

```diff
@@ -0,0 +1,272 @@
+<?php
+namespace Elementor\Core\Editor\Loader;
+
+use Elementor\Core\Utils\Collection;
+use Elementor\Plugin;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+abstract class Editor_Base_Loader implements Editor_Loader_Interface {
+	/**
+	 * @var Collection
+	 */
+	protected $config;
+
+	/**
+	 * @param Collection $config
+	 */
+	public function __construct( Collection $config ) {
+		$this->config = $config;
+	}
+
+	/**
+	 * @return void
+	 */
+	public function init() {
+		// Nothing to do.
+	}
```

Comment:
so why do you implement it?
pretty sure you don't have to implement methods in abstract class the implements interface